platforms/ios/lib/WysiwygComposer/Sources/WysiwygComposer/Tools/HTMLParser.swift

```diff
@@ -30,23 +30,48 @@ final class HTMLParser {
     ///   - html: HTML to parse
     ///   - encoding: string encoding to use
     ///   - textColor: text color to apply to the result string
+    ///   - codeBackgroundColor: color to apply to the background of code blocks
```

Comment:
could you add missing link color as well 🙏

Reply:
ahaha definitively